src/assets/translations/en-us.yaml

```diff
@@ -2112,7 +2112,7 @@ namespaceList:
 node:
   detail:
     detailTop:
-      containerRuntime: Container Runtime
+      containerEngine: Container Engine
```

Comment:
This is from Rancher Dashboard (not our string), so changing it is… kind of pointless.

(A good indication is that we never added _our_ strings to the Chinese translation.)